ogbot/core/defender.py

```diff
@@ -15,7 +16,12 @@ def auto_build_defenses(self):
         """Auto build defenses on all planets"""
         planets = self.planets
         for planet in planets:
-            self.auto_build_defenses_to_planet(planet)
+            try:
+                self.auto_build_defenses_to_planet(planet)
+
+            except Exception as e:
```

Comment:
I'd recommend removing the "as e" part of this if you aren't accessing the exception in the except codeblock.